src/pages/Categories/CategoriesBeta.vue

```diff
@@ -45,6 +45,20 @@
 					/>
 				</div>
 			</kv-grid>
+			<kv-grid class="tw-bg-secondary tw-pt-5 tw-mb-5 tw-rounded">
+				<div class="tw-flex tw-items-center tw-justify-center tw-pt-6 tw-pb-6">
+					<router-link
+						class="tw-text-black
+						active:tw-text-black visited:tw-text-black
+						hover:tw-text-black focus:tw-text-black"
+						to="/lend"
```

Comment:
please add analytics to this link we can use: `['Lending', 'click-view-all', 'View All']`